src/SWP/ContentBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,16 @@
+services:
+    swp_renderer.phpcr.initializer:
+        class: Doctrine\Bundle\PHPCRBundle\Initializer\GenericInitializer
+        arguments:
+            - SWP ContentBundle paths
+            - [ "/swp/content" ]
+        tags:
+            - { name: "doctrine_phpcr.initializer" }
+
+    swp_template_engine.loader.article:
+        class: SWP\ContentBundle\Loader\ArticleLoader
+        arguments:
+            - %kernel.root_dir%
+            - @doctrine_phpcr.odm.document_manager
```

Comment:
maybe we can set an alias for this service to shorter name ? its too way long imo, `document_manager` alias would be fine